lib/perl5/site_perl/HadoopTrimming.pm

```diff
@@ -11,6 +11,7 @@ use Utility;
 
 
 sub new {
+	die("OBSOLETE: HadoopTrimming::new shouldn't be called");
```

Comment:
@robmaz - should we just remove completely the code instead in all this methods? I realized that we have backed the code in the legacy branches, that we can retrieve anyway at some point...

Reply:
I wouldn't bother for now.